docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,330 @@
+
+``Datatable`` strives to mimic `Rdatatable's <https://rdatatable.gitlab.io/data.table/index.html>`_ API as closely as possible; however, there are differences due to language constraints.
+
+This page shows how to perform similar basic operations in `Rdatatable <https://rdatatable.gitlab.io/data.table/index.html>`_  in ``datatable``.
+
+Subsetting Rows
+----------------
+Sample data::
+
+    from datatable import dt, f, g, by, update, ifelse, join, sort
+    import numpy as np
+
+    # data is from the example data in Rdatatable
+
+    df = dt.Frame({"x":np.repeat(["b","a","c"],3),
+                   "y": [1,3,6]*3,
+                   "v": range(1,10)}) # datatable
+
+    DT = data.table(x=rep(c("b","a","c"),each=3),
+                    y=c(1,3,6), v=1:9) # rdatatable
+
+
+===============================   ============================================ ==================================================
+RDatatable                           Datatable                                   Action
+===============================   ============================================ ==================================================
+``DT[2]``                            ``df[1, :]``                               Select 2nd row
+``DT[2:3]``                          ``df[1:3, :]``                             Select 2nd and 3rd row
+``DT[3:2]``                         ``df[[2,1], :]``                            Select 3rd and 2nd row
+``DT[.N]``                          ``df[-1, :]``                               Select the last row
+``DT[y>2]``                         ``df[f.y>2, :]``                            All rows where ``y > 2``
+``DT[y>2 & v>5]``                   ``df[(f.y>2) & (f.v>5), :]``                Compound logical expressions
+``DT[!2:4]`` or ``DT[-(2:4)]``      ``df[[0, slice(4, None)], :]``              All rows other than rows 2,3,4
+``DT[order(x), ]``                  | ``df.sort("x")`` or                       Sort by column ``x``, ascending
+                                    | ``df[:, :, sort("x")]``
+``DT[order(-x)]``                   | ``df.sort(-f.x)`` or                      Sort by column ``x``, descending
+                                    | ``df[:, :, sort(-f.x)]``
+``DT[order(x, -y)]``                | ``df.sort(x, -f.y)`` or                   Sort by column ``x`` ascending, ``y`` descending
+                                    | ``df[:, :, sort(x, -f.y)]``
+===============================   ============================================ ==================================================
+
+Note the use of the ``f`` symbol when performing computations or sorting in descending order. You can read more about :ref:`f-expressions`.
+
+Selecting Columns
+------------------
+
+=========================================== ============================================ ==============================================
+RDatatable                                           Datatable                                                  Action
```

Comment:
The Python package name is better to spell as `datatable` everywhere on this tutorial, because this is how it is named on [pypi](https://pypi.org/project/datatable/).